exasol/exaslpm/pkg_mgmt/install_packages.py

```diff
@@ -1,6 +1,27 @@
 import pathlib
 
 import click
```

Comment:
I think it's not necessary to import click here.

Btw, if youupdate exasol-pytho-toolbox to v4, the nox task `nox -s format:fix` will remove automatically all unused imports.....